apm-sniffer/apm-toolkit-activation/apm-toolkit-log4j-2.x-activation/src/main/java/org/apache/skywalking/apm/toolkit/activation/log/log4j/v2/x/log/GRPCLogAppenderInterceptor.java

```diff
@@ -114,12 +116,23 @@ private LogData transform(final AbstractAppender appender, LogEvent event) {
             builder.setEndpoint(primaryEndpointName);
         }
 
-        return -1 == ContextManager.getSpanId() ? builder.build()
-                : builder.setTraceContext(TraceContext.newBuilder()
-                .setTraceId(ContextManager.getGlobalTraceId())
-                .setSpanId(ContextManager.getSpanId())
-                .setTraceSegmentId(ContextManager.getSegmentId())
-                .build()).build();
+        // fix -Dlog4j2.contextSelector=org.apache.logging.log4j.core.async.AsyncLoggerContextSelector not traceId
```

Comment:
This comment seems not current. You should explain about `AsyncLoggerContextSelector` is working on async mode, so it has to use `SkyWalkingContext`.

Comments for `fix xxx` actually is pointless, because git log already includes this information.